### PR TITLE
Fix module list

### PR DESCRIPTION
**Pull request by remusao**
@chrmod changed a few things to make Elm compile.

### CODE REVIEW

**Comment by chrmod** [comment]
@remusao have just pushed an update to that branch, please take a look on original PR

File: src/static/renderer.ts [hunk 1]
```diff
@@ -56,11 +57,11 @@ elmApp.ports.requestISocksProxy.subscribe(() => {
 
 const handleWebRequest = (data, respond) => {
   const message = JSON.parse(data);
-  const webRequest = CLIQZ.app.modules['webrequest-pipeline'].background;
+  const webRequest = cliqzApp.modules['webrequest-pipeline'].background;
   const eventName = message.functionName;
   console.log('received: ', message);
 
-  if (!webRequestEvents.includes(eventName) || !(eventName in webRequest)) {
```
Comment (chrmod) [inline src/static/renderer.ts:63]: why not `includes`?

Comment (remusao) [reply src/static/renderer.ts:63]: It's not supported by typescript by default I think. We need to target `es7`: https://github.com/Microsoft/TypeScript/issues/11890

Comment (chrmod) [reply src/static/renderer.ts:63]: as we run on node8 it should be fine to set that target

**Comment by chrmod** [comment]
closing in favor of https://github.com/remusao/broxy/pull/18

**Pull request closed without merging**